src/components/ContactInfo/ContactInfo.vue

```diff
@@ -341,9 +341,9 @@ export default class ContactInfo extends Vue {
         (v: any) => !v || (v.length === 0 || v.length === 14) || 'Phone number is invalid'
       ]
 
-    // Await the applied rules and validate form
-    await Vue.nextTick()
+    // Validate form and wait for v-model to get updated
     this.$refs.editContactForm.validate()
+    await Vue.nextTick()
```

Comment:
There is no need to wait to the rules to apply -- `validate()` handles that.

However, we do need to wait for prop `formValid` to get updated by validate.

Another way to do this would be to check the response from validate -- if True then the form is valid, else not.

Reply:
This bug caused the code to see the PREVIOUS value of `formValid`. In practise this means that the first time this method (submitContact) is run, the form is valid, but on subsequent calls it might be invalid, so updateContactInfo() is not called (line 349). The user has to click the OK button a second time.